arrow-test/src/main/kotlin/arrow/test/generators/Generators.kt

```diff
@@ -146,4 +147,13 @@ fun <K, V> genMap(genK: Gen<K>, genV: Gen<V>): Gen<Map<K, V>> =
         Gen.create { Gen.list(genK).generate().map { it to genV.generate() }.toMap() }
 
 fun <K, V> genMapKW(genK: Gen<K>, genV: Gen<V>): Gen<MapKW<K, V>> =
-        Gen.create { Gen.list(genK).generate().map { it to genV.generate() }.toMap().k() }
\ No newline at end of file
+        Gen.create { Gen.list(genK).generate().map { it to genV.generate() }.toMap().k() }
+
+class TimeUnitGen : Gen<TimeUnit> {
```

Comment:
Following the code style of the other generators, instead of a class this should be a function genTimeUnit() that returns an anonymous `object: Gen<TimeUnit>`